services/openuserjs/openuserjs-downloads.service.js

```diff
@@ -0,0 +1,27 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSDownloads extends BaseOpenUserJSService {
+  static category = 'downloads'
+  static route = { base: 'openuserjs', pattern: 'dt/:username/:scriptname' }
+
+  static examples = [
+    {
+      title: 'OpenUserJS',
+      namedParams: {
+        username: 'NatoBoram',
+        scriptname: 'YouTube_Comment_Blacklist',
+      },
+      staticPreview: renderDownloadsBadge({ downloads: 47 }),
```

Comment:
@NatoBoram Are you fine with your script being used as an example on shields.io? If not, I can change it.

Reply:
Oh, sure, have fun!